src/host/selectionInput.cpp

```diff
@@ -399,7 +399,7 @@ bool Selection::HandleKeyboardLineSelectionEvent(const INPUT_KEY_INFO* const pIn
         }
         case VK_PRIOR:
         {
-            coordSelPoint.Y -= sWindowHeight; // TODO: potential underflow
+            coordSelPoint.Y = base::CheckSub(coordSelPoint.Y, sWindowHeight).ValueOrDefault(bufferSize.BottomInclusive());

```

Comment:
should the minimum here be 0 instead of the inclusive bottom?

Reply:
I think it should be inclusiveTop